bin/remixd

```diff
@@ -1,46 +1,48 @@
 #!/usr/bin/env node
-var Router = require('../src/router')
-var servicesList = require('../src/servicesList')
+var { Router, services } = require('../src');
```

Comment:
In this file use `const` when possible. Else `let`

Reply:
Definitely, this is the old code, will update it.

Reply:
Resolving this as this is the old version that is no longer used